tests/details/lending-bar-auto-renew.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test } from '../fixtures';
+
+/**
+ * auto-renew feature testing starts here.......
+ */
+test(`Lending-Bar with auto-renew for 5 minutes`, async ({ lendingBarAutoRenew }) => {
+  const demoApp = 'https://internetarchive.github.io/iaux-book-actions/pr/pr-73/?timer=5';
```

Comment:
QUESTION: Is this a temporarily hard-coded link? It doesn't seem like a permanent value...

Reply:
Its temporary URL for these tests. Later we can change these tests with production when we ship auto-renew feature on archive.org